tests/test_client_helpers.py

```diff
@@ -0,0 +1,107 @@
+import random
+import string
+import uuid
+
+import pytest
+
+import lakefs_spec.client_helpers as client_helpers
+from lakefs_spec import LakeFSFileSystem
+from tests.util import RandomFileFactory
+
+
+def test_create_tag(
+    random_file_factory: RandomFileFactory, fs: LakeFSFileSystem, repository: str, temp_branch: str
+) -> None:
+    random_file = random_file_factory.make()
+    lpath = str(random_file)
+    rpath = f"{repository}/{temp_branch}/{random_file.name}"
+    fs.put(lpath, rpath, precheck=False)
+    client_helpers.commit(
+        client=fs.client, repository=repository, branch=temp_branch, message="Commit File Factory"
+    )
+    tag_name = f"Change_{uuid.uuid4()}"
+    client_helpers.create_tag(
+        client=fs.client, repository=repository, ref=temp_branch, tag=tag_name
+    )
+
+    assert any(
+        commit.get("id") == tag_name for commit in fs.client.tags_api.list_tags(repository).results
+    )
+
+
+def test_get_tags(fs: LakeFSFileSystem, repository: str) -> None:
+    response = client_helpers.get_tags(client=fs.client, repository=repository)
+    assert "pagination" in response
+    assert "results" in response
```

Comment:
This test does not make much sense - you are essentially testing `lakefs_client` code.

Maybe this can be integrated into the `test_create_tag` test?

Reply:
Yeah, was unsure about this as well. To be fair the `get_tags` function also does not do much more rather than calling the `lakefs_client` so there is not really much more to test. 

As with integration into the `test_create_tag` test, i think it would make more sense to keep the test logic of two separate functions separated.

Reply:
Then I would say let's drop the test for now.